src/genericComponents/Button.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import styled from 'styled-components';
+// import { colors, fonts } from '../theme/styleVars';
```

Comment:
Commenting this out will cause errors because you need the variables in your styles below